packages/@aws-cdk/aws-logs/test/loggroup.test.ts

```diff
@@ -149,17 +149,45 @@ describe('log group', () => {
 
   });
 
-  test('import from arn', () => {
+  test('import from arn, same region', () => {
     // GIVEN
     const stack2 = new Stack();
 
     // WHEN
-    const imported = LogGroup.fromLogGroupArn(stack2, 'lg', 'arn:aws:logs:us-east-1:123456789012:log-group:my-log-group');
+    const imported = LogGroup.fromLogGroupArn(
+      stack2,
+      'lg',
+      `arn:aws:logs:${stack2.region}:123456789012:log-group:my-log-group`,
+    );
     imported.addStream('MakeMeAStream');
 
     // THEN
     expect(imported.logGroupName).toEqual('my-log-group');
-    expect(imported.logGroupArn).toEqual('arn:aws:logs:us-east-1:123456789012:log-group:my-log-group:*');
+    expect(imported.logGroupArn).toEqual(`arn:aws:logs:${stack2.region}:123456789012:log-group:my-log-group:*`);
+    expect(imported.env.region).toEqual(stack2.region);
+    expect(stack2).toHaveResource('AWS::Logs::LogStream', {
+      LogGroupName: 'my-log-group',
+    });
+
+  });
```

Comment:
Can we leave this test unchanged?

Reply:
What exactly do you mean? A whole file or particular test case?

Reply:
@skinny85 are you referring to adding a test that verifies different region?

Reply:
I mean not making changes to the existing test case.

Reply:
Ah alright. I though that it will make much more sense to clearly separate those test cases.
Should I revert just the test body and title or just body and leave a title? Personally, I would keep the title but either way is fine with me.

PS. I will start from reverting the body of the first test leaving the title and if you prefer to also have it unchanged, let me know.

Reply:
Yeah, the name change is fine, let's leave the body as-is (= before this PR) though.

Reply:
Alright, done. Cheers 👍